lib/pollTravis.js

```diff
@@ -172,8 +178,27 @@ function createGhCommentFn (options) {
   }
 }
 
+function createGhStatusFn (options) {
+  const prInfo = prInfoStr(options)
+
+  return (state, travisId, message) => {
+    githubClient.statuses.create({
+      user: options.owner,
+      repo: options.repoName,
+      sha: options.lastSha,
+      target_url: `https://travis-ci.org/${options.owner}/${options.repoName}/builds/${travisId}`,
+      context: "Travis CI via nodejs-github-bot",
```

Comment:
`nodejs-github-bot/travis-ci` or something may be better, not sure. Not really critical.